### PR TITLE
Week 1 - Part 1

### PR DESCRIPTION
**Pull request by rein-vanimschoot**
This is the first required pull request of week 1. 
The tests work as intended.

### CODE REVIEW

**Comment by perrycarbone** [comment]
Hey Rein!

I reviewed both of your commits and didn't find any issues.  Everything looks good to me.

-Perry

File: app/models/category.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class Category < ActiveRecord::Base
-  has_many :videos
+  has_many :videos, -> { order("title")}
```
Comment (tomtomecek) [inline app/models/category.rb]: There is a space missing before `}`

File: spec/models/category_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe Category do
+  it { should have_many(:videos)}
```
Comment (tomtomecek) [inline spec/models/category_spec.rb]: There is a space missing before `}`

File: spec/models/video_spec.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe Video do
+  it { should belong_to(:category)}
+  it { should validate_presence_of(:title)}
+  it { should validate_presence_of(:description)}
```
Comment (tomtomecek) [inline spec/models/video_spec.rb]: There is a space missing before `}` in all 3 specs.

**Pull request merged**